modules/directives/src/main/scala/scala/build/preprocessing/directives/ScalaJs.scala

```diff
@@ -76,14 +83,37 @@ final case class ScalaJs(
       avoidLetsAndConsts = jsAvoidLetsAndConsts,
       moduleSplitStyleStr = jsModuleSplitStyleStr,
       esVersionStr = jsEsVersionStr,
-      noOpt = jsNoOpt
+      noOpt = jsNoOpt,
     )
-    BuildOptions(
-      scalaJsOptions = scalaJsOptions
+
+    def absFilePath(pathStr: String): Either[ImportMapNotFound, Path] = {
+      Try {
+        os.Path(pathStr)
+      }.orElse(
+        Try{
+          os.Path(pathStr, base = os.pwd)
+        }
+      ).toEither.fold(ex =>
```

Comment:
```suggestion
      Try(os.Path(pathStr, os.pwd)).toEither.fold(ex =>
```

Reply:
I originally had the idea, to be able to specify a file at any location in a sfilesystem. I think that isn't a good idea, and that the import map shoudl be part of a repository. So agreed.